huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -68,12 +69,59 @@ var AdvisorAssignmentsView = React.createClass({
               {this.renderAssignmentRows()}
             </table>
           </div>
-          <div className="tablemenu footer" />
+          <div className="tablemenu footer">
+            <Button
+              color="green"
+              size="small"
+              loading={this.state.loading}
+              onClick={this._handleClick}>
+              Finalize Assignments
+            </Button>
+          </div>
         </form>
       </InnerView>
     );
   },
 
+  _handleClick: function(event) {
+    this.setState({loading: true});
+    var user = this.props.user.getData();
+    var school = this.props.user.getSchool();
+    var elements = document.getElementsByTagName("input");
```

Comment:
The whole point of React is to abstract the DOM away. Don't do this. Instead, whatever data you want to send should be inside this component's `state`. A component's props and state are its source of truth: the `render` function uses that state to build the UI, and if you need to send data to the server, that's where it comes from.